tox.ini

```diff
@@ -30,4 +30,4 @@ setenv =
    LC_ALL=en_US.UTF-8
 commands =
     flake8 pinax
-    coverage run setup.py test
+    coverage run setup.py test {posargs}
```

Comment:
Added to https://github.com/pinax/pinax-stripe/pull/355/commits/f717fbee62606415550defdc16038791c7ac6308.